gatsby-node.js

```diff
@@ -275,6 +275,12 @@ exports.createPages = async ({ actions, graphql, reporter }) => {
     redirectInBrowser: true,
     isPermanent: true,
   });
+  createRedirect({
```

Comment:
why a single blog needed a redirect. This seems a redundant change. We should avoid this and instead changed the path where this blog is been mentioned if its being used in less places.